clang/test/OpenMP/depobj_codegen.cpp

```diff
@@ -36,6 +36,8 @@ int main(int argc, char **argv) {
 #pragma omp depobj(b) update(mutexinoutset)
 #pragma omp depobj(a) depend(iterator(char *p = argv[argc]:argv[0]:-1), out: p[0])
   (void)tmain(a), tmain(b);
+   omp_depend_t obj;
```

Comment:
Auto-gen check lines please. You might want do do a pre-commit for existing test.

Reply:
updated !!!